app/Schema/account_transaction_details_schema.py

```diff
@@ -48,16 +47,15 @@ class FundTransferSchema(ma.Schema):
 
     """Adding the Schema validations to the FundTransfer """
 
-    from_account = fields.Str(required=True, validate=Regexp("^[0-9]{10}$|^[0-9]{12}$"))
-    to_account = fields.Str(validate=Regexp("^[0-9]{10}$|^[0-9]{12}$"))
-    transaction_amount = fields.Int(required=True)
+    from_account = fields.Str(required=True, validate=(Regexp("^[0-9]{10}$|^[0-9]{12}$"), Length(equal=10)))
+    to_account = fields.Str(validate=(Regexp("^[0-9]{10}$|^[0-9]{12}$"), Length(equal=10)))
+    transaction_amount = fields.Int(strict=True, required=True, validate=Range(min=1000, max=50000))
 
     class Meta:
 
         """ Exposed fields """
 
         fields = ('id', 'from_account', 'to_account', 'transaction_amount')
-        unknown = INCLUDE
```

Comment:
What does `INCLUDE` does?

Reply:
If we entered the field name incorrect it doesn't  show the "UNKNOWN field error" , And if we entered any other field which we have not taken in the table it does not gives any error their.